go.mod

```diff
@@ -226,6 +227,7 @@ require (
 	github.com/prometheus/client_model v0.2.0 // indirect
 	github.com/prometheus/common v0.31.1 // indirect
 	github.com/prometheus/procfs v0.7.3 // indirect
+	github.com/rakyll/statik v0.1.7 // indirect
```

Comment:
we should no longer need `statik`

Reply:
Hi @aaron-prindle , this is what happened when I tried to run the tests:
```
skaffold$ make test
go test -count=1 -race -short -timeout=90s ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
cmd/skaffold/app/cmd/statik/statik.go:7:2: no required module provides package github.com/rakyll/statik/fs; to add it:
	go get github.com/rakyll/statik/fs

=== Slow Tests ===
exit status 1
make: *** [Makefile:112: test] Error 1

skaffold$ go get github.com/rakyll/statik/fs
go: added github.com/rakyll/statik v0.1.7

skaffold$ make test
go test -count=1 -race -short -timeout=90s ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/apiversion	0.184s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/bazel	0.399s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/buildpacks	0.573s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/custom	0.316s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/docker	0.764s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache	0.503s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build	1.052s
           ...
```

Am I missing something?

Reply:
Im a bit confused why that error occurred as statik.go was removed
https://github.com/GoogleContainerTools/skaffold/pull/7339/files#diff-1b85c990747d535d0dc3f220994e9d1eb32662ebe38aa052e60bc15386763d71

```
aprindle@aprindle ~/skaffold  [main]$ find . -name statik.go # no output
```